netlify/edge-functions/og-image.tsx

```diff
@@ -0,0 +1,262 @@
+                fill="#007FFF"
+              />
+            </svg>
+            <div
+              style={{
+                height: 40,
+                width: 2,
+                backgroundColor: '#DAE2ED',
+                margin: '0 24px',
+              }}
+            />
+            <p
+              style={{
+                fontFamily: 'General Sans',
+                fontSize: '24px',
+                fontWeight: 600,
+                lineHeight: '40px',
+                color: '#007FFF',
+              }}
+            >
+              {product}
+            </p>
+          </div>
+          <div
+            style={{
+              fontFamily: 'General Sans',
+              fontStyle: 'normal',
+              fontWeight: 600,
+              fontSize: '68px',
+              lineHeight: '82px',
+              color: '#0B0D0E',
+              display: 'flex',
+              flexDirection: 'column',
+              justifyContent: 'center',
+            }}
+          >
+            {title &&
+              title.split('\\n').map((line) => (
+                <p style={{ margin: 0, flexWrap: 'wrap' }}>
+                  {line.split('*').flatMap((text, index) => {
+                    if (index > 0) {
+                      starCount += 1;
+                    }
+
+                    const isBlue = starCount % 2 === 1;
+                    return text.split(' ').map((word) => (
+                      <span
+                        style={{
+                          color: isBlue ? '#007FFF' : '#0B0D0E',
+                          marginRight: word.length > 0 ? 15 : 0,
+                        }}
+                      >
+                        {word}
+                      </span>
+                    ));
+                  })}
+                </p>
+              ))}
+            {description && (
+              <p
+                style={{
+                  fontSize: '24px',
+                  fontWeight: 600,
+                  lineHeight: '40px',
+                  marginTop: 20,
```

Comment:
```suggestion
                  fontFamily: 'IBM Plex Sans',
                  fontSize: '36px',
                  fontWeight: 600,
                  color: '#434D5B',
                  lineHeight: '58px',
                  marginTop: 24,
```

Already taking into consideration having the IBM font. Here are some style tweaks to the description text.